modules/smart-agent_elasticsearch/conf/01-cluster-status.yaml

```diff
@@ -0,0 +1,19 @@
+module: ElasticSearch
+name: "cluster status"
+aggregation: true
+transformation: ".mean(over='5m')"
```

Comment:
Why do a mean on 5 minutes since thresholds are strict comparison and lasting is already set ?